### ISSUE TITLE
CloudWatch queries can no longer be disabled (error + empty graph) 

### ISSUE THREAD

**Issue by cykl** [grafana/grafana#17112]
**What happened**:

While testing 6.2.0-beta2 I discovered a regression in some panels using a CloudWatch data source. Graph panels show an error when a query with an id is disabled.

Since CloudWatch introduced math expressions, it is common to have disabled queries. The first query is only used by the second query and is not intended to be displayed. Prime example is likely graphing a request rate.

![image](https://user-images.githubusercontent.com/1764725/57843749-1ffb2880-77cf-11e9-9068-889712144c9e.png)

Tooltip error says "_Cannot read property 'series' of undefined_" and console shows:

```
metrics_panel_ctrl.ts:179 Panel data error: TypeError: Cannot read property 'series' of undefined
    at datasource.ts:194
    at angular.js:17051
    at angular.js:17095
    at l.$digest (angular.js:18232)
    at l.$apply (angular.js:18531)
    at angular.js:18832
    at h (angular.js:6362)
    at angular.js:6642
```

**What you expected to happen**:

Disabled query are not displayed. Enabled queries are displayed. 

**How to reproduce it (as minimally and precisely as possible)**:

![image](https://user-images.githubusercontent.com/1764725/57843439-94819780-77ce-11e9-8cb9-51eb3b4de5cc.png)

Error disappears when I re-enable the query or remove the id.

**Anything else we need to know?**:

This used to work with Grafana 6.0.0. I haven't tried 6.1.x yet.

**Environment**:
- Grafana version: 6.2.0-beta2
- Data source type & version: CloudWatch 
- OS Grafana is installed on: Ubuntu 14.04 LTS
- User OS & Browser: Firefox & Chrome on Windows 7
- Grafana plugins: None

**Comment by cykl** [comment]
I Just gave 6.1.6 as shoot. It works fine.

**Comment by jpiccari** [comment]
I took a look and this was broken by #16408. Specifically [this line](https://github.com/grafana/grafana/pull/16408/files#diff-5be6ea1dd05a4db39f000439bbca4f6aR142) results in `queryRes` being set to `undefined`, when it is used on the next line to access `series` an exception is thrown.

I'll open a PR in a bit.

**Comment by pcolby** [comment]
This has broken more than half of my dashboards!! (We use a lot of compound CW metrics)

I have to say, I'm very disappointed that 6.2 was released (and now pushed to my (paid) Grafana Cloud instance, with no way for me to rollback) with such a critical regression known, and not yet resolved.

**Comment by torkelo** [comment]
Sorry , it should be fixed in stable , if not we will investigate and fix asap

**Comment by torkelo** [comment]
Sorry that this bug was still present in 6.2 stable, we had a related bug that was fixed in 6.2 stable https://github.com/grafana/grafana/pull/17124 that was affecting all disabled queries, but it appears that this was not caused by that but by a Cloudwatch change. 

PR with fixed is merge, https://github.com/grafana/grafana/pull/17283 . 6.2.2 patch release should be out soon.
